src/current/_includes/common/upgrade-cockroach-version-license-limitations.md

```diff
@@ -0,0 +1,22 @@
+To perform major version upgrades, you must have [a valid license key]({% link {{ page.version.version }}/licensing-faqs.md %}#obtain-a-license).
+
+Patch version upgrades can be performed without a valid license key, with the following limitations:
+
+- The cluster will run without limitations for a specified grace period. During that time, alerts are displayed that the cluster needs a valid license key. For more information, see the [Licensing FAQs]({% link {{ page.version.version }}/licensing-faqs.md %}).
```

Comment:
changed to "refer to"

"specified" refers to that which is specified by the alerts that are displayed (which are explained in detail in the linked Licensing FAQs).

Based on convo with @spilchen my previous specific day number was going to create confusion because the # of days is different depending on whether you are upgrading from an earlier Core release vs. a new post-CSL release.  That logic branch is hard to describe properly here and would also duplicate the already existing description in the updated Licensing FAQs page.

it's ... not my fave, but a lot of this licensing/throttling language is pretty tortured and IMO a lot of it boils down to "just keep your license up to date yo" but i'm just a caveman, this modern world frightens and confuses me 🤷